src/Compiler/FSComp.txt

```diff
@@ -1665,4 +1665,5 @@ reprStateMachineInvalidForm,"The state machine has an unexpected form"
 3549,tcSynTypeOrInvalidInDeclaration,"SynType.Or is not permitted in this declaration"
 3550,chkDuplicatedMethodParameter,"Duplicate parameter. The parameter '%s' has been used more that once in this method."
 featureEscapeBracesInFormattableString,"Escapes curly braces before calling FormattableStringFactory.Create when interpolated string literal is typed as FormattableString"
-3551,chkErrorOnStaticClasses,"if a type uses both [<Sealed>] and [<AbstractClass>] it means it is static. No %s allowed"
\ No newline at end of file
+3551,buildDuplicateFile,"The source file '%s' (at position %d/%d) already appeared in the compilation list (at position %d/%d). Please verify that it is included only once in the project file."
+3552,chkErrorOnStaticClasses,"if a type uses both [<Sealed>] and [<AbstractClass>] it means it is static. No %s allowed"
```

Comment:
```suggestion
3552,chkErrorOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. %s is not allowed."
```

Reply:
I have updated to use 2 different error numbers so this is not longer the case :)